perf/tuple.cc

```diff
@@ -276,7 +276,7 @@ tuple_access_members(benchmark::State& state)
 			i = 0;
 		}
 		struct tuple *t = tuples[i++];
-		benchmark::DoNotOptimize(bool(t->is_dirty));
+		benchmark::DoNotOptimize(uint8_t(t->flags));
```

Comment:
Please don't change logic in perf test.. There must be corresponding `tuple_has_flag(tuple, TUPLE_IS_DIRTY)`

Reply:
Ok, replaced with `tuple_has_flag` and left corresponding comment.